test/Ocelot.AcceptanceTests/ConfigurationReloadTests.cs

```diff
@@ -88,7 +87,6 @@ public void should_not_trigger_change_token_with_no_change()
         public void Dispose()
         {
             _steps.Dispose();
-            GC.SuppressFinalize(this);
```

Comment:
Why did you remove the GC call? Because there is no descendants of the class, and there is no finalizer?
The `GC.SuppressFinalize` makes sense for subclasses and non-sealed class.
So, why not to mark the class as `sealed`? 😉

Reply:
ok for sealed